enigma-swing/src/main/java/org/quiltmc/enigma/gui/element/NavigatorPanel.java

```diff
@@ -18,19 +18,20 @@
 import java.util.List;
 import java.util.Map;
 
+// todo occasionally disappears for no reason
```

Comment:
pretty sure this could be fixed via keeping only one instance and properly removing/readding it everywhere